packages/app-store/routing-forms/pages/forms/[...appPages].tsx

```diff
@@ -111,182 +112,185 @@ export default function RoutingForms({
   ];
 
   return (
-    <ShellMain
-      heading="Routing Forms"
-      CTA={hasPaidPlan && forms?.length ? <NewFormButton /> : null}
-      subtitle={t("routing_forms_description")}>
-      <UpgradeTip
-        title={t("teams_plan_required")}
-        description={t("routing_forms_are_a_great_way")}
-        features={features}
-        background="/tips/routing-forms"
-        isParentLoading={<SkeletonLoaderTeamList />}
-        buttons={
-          <div className="space-y-2 rtl:space-x-reverse sm:space-x-2">
-            <ButtonGroup>
-              <Button color="primary" href={`${WEBAPP_URL}/settings/teams/new`}>
-                {t("upgrade")}
-              </Button>
-              <Button color="minimal" href="https://go.cal.com/teams-video" target="_blank">
-                {t("learn_more")}
-              </Button>
-            </ButtonGroup>
-          </div>
-        }>
-        <FormActionsProvider appUrl={appUrl}>
-          <div className="-mx-4 md:-mx-8">
-            <div className="mb-10 w-full px-4 pb-2 sm:px-6 md:px-8">
-              <div className="flex">
-                <TeamsFilter />
-              </div>
-              <FilterResults
-                queryRes={queryRes}
-                emptyScreen={
-                  <EmptyScreen
-                    Icon={GitMerge}
-                    headline={t("create_your_first_form")}
-                    description={t("create_your_first_form_description")}
-                    buttonRaw={<NewFormButton />}
-                  />
-                }
-                noResultsScreen={
-                  <EmptyScreen
-                    Icon={GitMerge}
-                    headline={t("no_results_for_filter")}
-                    description={t("change_filter_common")}
-                  />
-                }
-                SkeletonLoader={SkeletonLoaderTeamList}>
-                <div className="bg-default mb-16 overflow-hidden">
-                  <List data-testid="routing-forms-list">
-                    {forms?.map(({ form, readOnly }) => {
-                      if (!form) {
-                        return null;
-                      }
+    <LicenseRequired>
```

Comment:
SingleForm is used by form-edit, reporting, route-builder. So, we can add LicenseRequired there itself and we won't need to duplicate this everywhere.

Reply:
reverted LicenseRequired from `routing-forms/pages/forms`

Reply:
I am not sure why you reverted `/forms`. I wanted you to revert form-edit, reporting, route-builder because all of them use a common wrapper component `SingleForm` which we can wrap in LicenseRequired. It would automatically add LicenseRequired to form-edit, reporting and route-builder 

![Screenshot 2023-07-06 at 10.17.25 AM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/QOdRMCCW7wQwTOiuNnjo/a222f143-524d-4207-ad17-c3e1c00db5c3/Screenshot%202023-07-06%20at%2010.17.25%20AM.png)

Reply:
I see `SingleForm` used only with form-edit, reporting, and route-builder so wrapping `SingleForm` in `LicenseRequired` works for all three routes.

and `/forms` should be wrapped in `LicenseRequired`  right?

@hariombalhara

Reply:
Yeah exatly.